src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/UserService.kt

```diff
@@ -57,8 +57,14 @@ class UserService(
     val qualifications = application.getRequiredQualifications()
 
     while (true) {
+      var errorMessage: String
```

Comment:
Might be worth putting the user who created the application too to save us having to go into the DB to figure it out?  `application.createdByUser.deliusUsername`